Спринт 23. Домашнее задание

ignatov-ai opened the pull request:
Финал приложения

ignatov-ai committed:
Спринт 23.
Финал приложения

DmitrijEgorow commented:
Экран не соответствует дизайну: цвет текста не серый, боковые кнопки также должны иметь другой цвет

![Screenshot_20240502_004916](https://github.com/ignatov-ai/PlaylistMaker/assets/36837067/4e06d557-198a-4576-8754-2dcd6cdc6ee0)

DmitrijEgorow commented:
- [ ] Разметка не адаптируется под различные диагонали устройств
- [ ] Боковые кнопки светлее, чем у дизайнера

https://github.com/ignatov-ai/PlaylistMaker/assets/36837067/9b706aed-99f4-4d3c-91b5-0f35350c0bb9

DmitrijEgorow commented:
Черный текст плохо читаем на сером фоне
![Screenshot_20240502_004633](https://github.com/ignatov-ai/PlaylistMaker/assets/36837067/a434e641-ef87-4d96-b89a-7efcbc69d5b9)

DmitrijEgorow commented:
Нижнего меню навигации на этом экране быть не должно

![Screenshot_20240502_004604](https://github.com/ignatov-ai/PlaylistMaker/assets/36837067/765ae359-0e28-427d-959f-ab9ad30ad246)

DmitrijEgorow commented:
- [ ] Нижнего меню навигации на этом экране быть не должно
- [ ] На устройствах с увеличением текста наблюдается обрезка по вертикали
- [ ] Что-то не влезает на данном экране
![Screenshot_20240502_004721](https://github.com/ignatov-ai/PlaylistMaker/assets/36837067/a04be88c-3a36-4683-bec9-c4695588a2e5)

DmitrijEgorow commented:
- [ ] Точка не центрирована, должна быть чёрного цвета
- [ ] Я бы убрал выделение у текущей выбранной вкладки нижнего меню

![Screenshot_20240502_004803](https://github.com/ignatov-ai/PlaylistMaker/assets/36837067/a55aef7f-0bae-4fce-ad38-987b6c335484)

DmitrijEgorow commented on app/src/main/res/navigation/main_navigation_graph.xml:
```diff
@@ -39,6 +39,9 @@
             android:id="@+id/action_mediaFragment3_to_newPlaylistFragment"
             app:destination="@id/newPlaylistFragment" />
+        <action
+            android:id="@+id/action_mediaFragment3_to_playlistShowFragment"
```
Может, убрать 3?

ignatov-ai committed:
Спринт 23.
Исправлена часть замечаний

ignatov-ai committed:
Спринт 23.
Исправлена часть замечаний

ignatov-ai committed:
Спринт 23.
Исправлена часть замечаний

ignatov-ai committed:
Спринт 23.
Исправлена часть замечаний

ignatov-ai committed:
Спринт 23.
Исправлена часть замечаний

ignatov-ai commented:
Не понял почему стоят знаки вопросов у пунктов "Поделиться", "Редактировать"

ignatov-ai committed:
Спринт 23.
Исправлена часть замечаний

ignatov-ai committed:
Спринт 23.
Исправлена часть замечаний

ignatov-ai committed:
Спринт 23.
Исправлена часть замечаний

DmitrijEgorow commented:
> Не понял почему стоят знаки вопросов у пунктов "Поделиться", "Редактировать", "Удалить" и т.п.

До этих кнопок нельзя добраться, чтобы посмотреть, что они работают

DmitrijEgorow commented:
> * [x]  Точка не центрирована, должна быть чёрного цвета
> * [x]  Я бы убрал выделение у текущей выбранной вкладки нижнего меню
> 
> ![Screenshot_20240502_004803](https://private-user-images.githubusercontent.com/36837067/327197395-a55aef7f-0bae-4fce-ad38-987b6c335484.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ2MTMyNTQsIm5iZiI6MTcxNDYxMjk1NCwicGF0aCI6Ii8zNjgzNzA2Ny8zMjcxOTczOTUtYTU1YWVmN2YtMGJhZS00ZmNlLWFkMzgtOTg3YjZjMzM1NDg0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTAyVDAxMjIzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ2N2E2NWVhZGQ1ZDdkZGU0MjJkOTM3YzkyMTBkOWUxMzBjMTQ4YTQzYTQ4ZGNjYWM5ODc1NDU4OGRmZGM1MGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.CZeKFmY6O-UUfWsibH7W_BgS-8Ox_CojPHlfCt-oRZM)

рисовалось вот так

DmitrijEgorow commented:
Изменение обложки плейлиста не поддерживается

https://github.com/ignatov-ai/PlaylistMaker/assets/36837067/032a687c-896f-4781-8d3f-a61a8c2b5b33

DmitrijEgorow commented:
- [ ] Если в плейлисте нет добавленных треков, то меню должно исчезать
- [ ] Текста нет

https://github.com/ignatov-ai/PlaylistMaker/assets/36837067/5691cb91-a957-4734-a429-52beaff5490d

DmitrijEgorow commented:
- [ ] Часть текста должна иметь чёрный цвет
- [ ] Наложение текста нужно убрать
![Screenshot_20240502_161927](https://github.com/ignatov-ai/PlaylistMaker/assets/36837067/a5e7dd41-5377-4720-a630-c2abe2f54774)

DmitrijEgorow commented:
> Черный текст плохо читаем на сером фоне ![Screenshot_20240502_004633](https://private-user-images.githubusercontent.com/36837067/327196766-a434e641-ef87-4d96-b89a-7efcbc69d5b9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ2NjAxMjUsIm5iZiI6MTcxNDY1OTgyNSwicGF0aCI6Ii8zNjgzNzA2Ny8zMjcxOTY3NjYtYTQzNGU2NDEtZWY4Ny00ZDk2LWI4OWEtN2VmY2JjNjlkNWI5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTAyVDE0MjM0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTllZDc4MjBlYmQyMDhiZjc0ZDQ2YmZlMTI1OTZkZWNkZTZhY2I2YTZjZWZiMWU3ZGNmMGNmZGZlOTNhYjMwZTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Fx1jqWqRfUoc4qek-xk9Ms42wSbhPciZStS1BVcueLg)

Актуально

DmitrijEgorow commented:
![Screenshot_20240502_161740](https://github.com/ignatov-ai/PlaylistMaker/assets/36837067/ef5e1497-60c2-4d7e-97b1-f2b9310903de)

DmitrijEgorow commented:
Поделиться и три точки не влезли
![Screenshot_20240502_161725](https://github.com/ignatov-ai/PlaylistMaker/assets/36837067/f7cee5ca-bcea-45ca-b3e6-2750bd16e227)

DmitrijEgorow commented:
"T2" и "0 треков" должны быть белого цвета

![Screenshot_20240502_161628](https://github.com/ignatov-ai/PlaylistMaker/assets/36837067/5a21aa35-d5c5-48b0-ba56-6d09107109a9)

DmitrijEgorow commented:
Название плейлиста должно быть чёрным
![Screenshot_20240502_161539](https://github.com/ignatov-ai/PlaylistMaker/assets/36837067/0a883069-5648-46a5-bc3f-61b7c12f1605)

ignatov-ai committed:
Спринт 23.
Исправлена часть замечаний v2.0

ignatov-ai committed:
Спринт 23.
Исправлена часть замечаний v2.0

ignatov-ai committed:
Спринт 23.
Исправлена часть замечаний v2.0

ignatov-ai committed:
Спринт 23.
Исправлена часть замечаний v2.0 сделал динамическую высоту списка треков в плейлисте

ignatov-ai committed:
Спринт 23.
Исправлена часть замечаний v2.0

ignatov-ai commented:
> * [ ]  Если в плейлисте нет добавленных треков, то меню должно исчезать
> * [ ]  Текста нет
> 
>  Screen_recording_20240502_162434.mp4

не понял данного замечания, видео не проигрывается

ignatov-ai committed:
Спринт 23.
Исправлена часть замечаний v2.0 исправил ошибку с редактированием изображдения плейлиста.

Комментарий к замечанию в ПР:
 - Если в плейлисте нет добавленных треков, то меню должно исчезать
 - Текста нет
Screen_recording_20240502_162434.mp4

не понял данного замечания, видео не проигрывается, в макетах и ТЗ спринта ничего подобного не заметил

ignatov-ai commented:
Нашел пункт в ТЗ про исчезающее меню, хотя это немного не логично, ведь я мог промазать, а на самом деле хотел отредактировать, например, название плейлиста. При таком условии в ТЗ необходимо снова вызывать меню и нажимать на редактировать

ignatov-ai committed:
Спринт 23.
Исправлена часть замечаний v2.0
Добавил исчезновение меню при попытке поделиться пустым плейлистом

Нашел пункт в ТЗ про исчезающее меню, хотя это немного не логично, ведь я мог промазать, а на самом деле хотел отредактировать, например, название плейлиста. При таком условии в ТЗ необходимо снова вызывать меню и нажимать на редактировать

ignatov-ai committed:
Спринт 23.
Исправлена часть замечаний v2.0
Добавил исчезновение меню при попытке поделиться пустым плейлистом

Нашел пункт в ТЗ про исчезающее меню, хотя это немного не логично, ведь я мог промазать, а на самом деле хотел отредактировать, например, название плейлиста. При таком условии в ТЗ необходимо снова вызывать меню и нажимать на редактировать

DmitrijEgorow commented:
> * [x]  Разметка не адаптируется под различные диагонали устройств
> * [x]  Боковые кнопки светлее, чем у дизайнера
> 
>  Screen_recording_20240502_004700.mp4

Про адаптивность экрана плеера: давай сделаем либо сжатие размеров обложки, либо добавим ScrollView

ignatov-ai commented:
> ScrollView

Хорошо, чуть позже добавлю ScrollView

ignatov-ai committed:
Спринт 23.
Исправлена часть замечаний v2.0
Добавил ScrollView на экран плеера

ignatov-ai commented:
> > ScrollView
> 
> Хорошо, чуть позже добавлю ScrollView

Добавил, вроде работает)))
